go/multipooler/manager/manager.go

```diff
@@ -325,6 +326,55 @@ func (pm *MultiPoolerManager) GetMultiPooler() (*topo.MultiPoolerInfo, ManagerSt
 	return pm.multipooler, pm.state, pm.stateError
 }
 
+// GetBackupConfigPath returns the path to the pgbackrest config file
+func (pm *MultiPoolerManager) GetBackupConfigPath() string {
+	return filepath.Join(pm.config.PoolerDir, "pgbackrest.conf")
+}
+
+// GetBackupStanza returns the pgbackrest stanza name
+func (pm *MultiPoolerManager) GetBackupStanza() string {
+	// Use configured stanza name if set, otherwise fallback to service ID
+	if pm.config.PgBackRestStanza != "" {
+		return pm.config.PgBackRestStanza
+	}
+	return pm.serviceID.Name
+}
+
+// GetPgCtldClient returns the pgctld gRPC client
+func (pm *MultiPoolerManager) GetPgCtldClient() pgctldpb.PgCtldClient {
+	return pm.pgctldClient
+}
+
+// GetTableGroup returns the table group from the multipooler record
+func (pm *MultiPoolerManager) GetTableGroup() string {
+	pm.mu.Lock()
+	defer pm.mu.Unlock()
+	if pm.multipooler != nil && pm.multipooler.MultiPooler != nil {
+		return pm.multipooler.TableGroup
+	}
+	return ""
+}
```

Comment:
This can block if the Pooler is busy doing something that requires a lock. Vitess solves this by keeping a copy that can be read without locking.
https://github.com/vitessio/vitess/blob/main/go/vt/vttablet/tabletmanager/tm_state.go#L487:L490